contracts/jobs/JobsManager.sol

```diff
@@ -354,8 +354,6 @@ contract JobsManager is ManagerProxyTarget, IVerifiable, IJobsManager {
         Job storage job = jobs[_jobId];
         Claim storage claim = job.claims[_claimId];
 
-        // Job cannot be inactive
-        require(jobStatus(_jobId) != JobStatus.Inactive);
```

Comment:
I can't immediately think of any issues that might arise from a transcoder being able to call `verify()` for an inactive job - the benefit here is that it solves the problem of a transcoder leaving money on the table if the current block is very close to the job's end block and it wants to claim as many segments as possible by waiting until the very end - this would allow the transcoder to submit a claim at the very end, and then submit for verification after the job's end block

Reply:
Is it weird if they call verify after the slashing period has ended? So for example call `claim()` at the end of the round, and then not call `verify()` for 1000 blocks? I assume they would already be slashed but would any code here screw up the state if executed later?

Reply:
I think it is assumed that if the transcoder misses verification because it does not call `verify()` for a long time that a watcher will slash the transcoder during the verification slashing period. If the transcoder is not slashed, calls `verify()` many blocks later and passes verification, nothing should happen because a successful verification only fires an event

Reply:
Ah, I see. So since a successful verificaiton is equivalent to a lack of slashing during the slash period, there's actually no benefit to calling verify after the fact. Sounds good. :ship: